pkgs/applications/misc/procmail/default.nix

```diff
@@ -33,7 +33,7 @@ stdenv.mkDerivation rec {
 
   meta = with lib; {
     description = "Mail processing and filtering utility";
-    homepage = "http://www.procmail.org/";
+    homepage = "https://github.com/BuGlessRB/procmail";
```

Comment:
not sure about this

Reply:
This commit removes references to procmail.org for source code location: https://github.com/BuGlessRB/procmail/commit/0f184bf97b91600b2ab87dc9c0567014968fb53d however there is indeed no mention of the official homepage. 
Since the procmail.org has been showing the same empty page for 2 years: https://web.archive.org/web/20200815041923/http://www.procmail.org/ I thought github link would be more helpful to package users. Please note also that the mailing-lists currently indicated in the README as being @procmail.org are also nonfunctional.

Reply:
There is no way to indicate this. If we you want to use another sendmail add it to your path instead.

Reply:
@SuperSandro2000 The package providing sendmail should be there during build time. So is the code correct as-is? What other modifications should I include for this package?

Reply:
> The package providing sendmail should be there during build time.

Is there a reason in the program for that? Does it even matter what sendmail we are using or is there a good implementation that works in almost all use cases.

Other than this I think this is good to go if this is the new procmail authorative source.